.github/workflows/ci-sanitizers.yml

```diff
@@ -0,0 +1,128 @@
+name: CI with sanitizers enabled
+
+on:
+  # TODO: Enable pull request and cron triggers later, when this workflow is tested and not failing
```

Comment:
>TODO:  [](http://example.com/codeflow?start=4&length=6)

We do not allow code with TODO comments to be checked in main. Please complete this part now, or remove the comment and create a feature task for the team.

Reply:
OK, as in the comment, this cannot be done now since the workflow must first appear in the main branch. pull_request and schedule will be done in a separate PR and I'll drop this comment for now.